versions.yaml

```
@@ -17,6 +17,7 @@ module-sets:
     version: v1.0.0-rcv0011
     modules:
       - go.opentelemetry.io/collector/pdata
+      - go.opentelemetry.io/collector/featuregate
```

Let's keep it in alphabetical order?
```suggestion
      - go.opentelemetry.io/collector/featuregate
      - go.opentelemetry.io/collector/pdata
```